### PR TITLE
Apply horizontal session item design

### LINKED ISSUE #140: [Assigned] Apply HorizontalSessionItem design
## Overview (Required)
- It is not applied design
## Screenshot
| Current | Design|
|---|---|
| image | image |

### PR DESCRIPTION
**Pull request by kawmra**
## Issue
- close #140 

## Overview (Required)
- Apply horizontal session item design
  - Remove the level text view
  - Remove the divider view
  - Add a day number view
  - Add end ellipsize to the title

## Links
None

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt [hunk 1]
```diff
@@ -15,6 +15,17 @@ import io.github.droidkaigi.confsched2018.model.toReadableDateTimeString
 import io.github.droidkaigi.confsched2018.model.toReadableTimeString
 import java.util.regex.Pattern
 
+@BindingAdapter(value = ["bind:dayNumber", "bind:suffix"])
+fun TextView.setDayText(dayNumber: Int?, suffix: String?) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt]: These two args should be non-null, right?
If so, can you remove `?`?

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt [hunk 2]
```diff
@@ -15,6 +15,17 @@ import io.github.droidkaigi.confsched2018.model.toReadableDateTimeString
...
+    dayNumber ?: return
+    suffix ?: return
+    text = context.getString(
+            R.string.day_number_format,
+            dayNumber,
+            suffix
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt]: I think it's better to separate TextViews for day number and slash.
What do you think?

**Comment by konifar** [comment]
Sorry 🙇 Can you resolve the conflicts?

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt (hunk 1 above)
Comment (kawmra) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt]: Indeed.
I'll fix that. Thanks!

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt (hunk 2 above)
Comment (kawmra) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt]: A [similar code](https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/res/layout/item_horizontal_session.xml#L53) has `/` that defined as prefix. so I copied that pattern.

but I agree your suggestion. I'll fix it as well!

**Comment by kawmra** [comment]
got it 🙆

**Comment by kawmra** [comment]
@konifar 
sorry for my late reply.
I fixed it! please review 🙏

**Comment by konifar** [comment]
@kawmra It's okay! Sure! 👀

File: app/src/main/res/layout/item_horizontal_session.xml [hunk 3]
```diff
@@ -63,34 +84,16 @@
...
                 android:id="@+id/title"
                 android:layout_width="0dp"
                 android:layout_height="62dp"
                 android:layout_marginEnd="16dp"
                 android:layout_marginStart="12dp"
                 android:layout_marginTop="10dp"
+                android:ellipsize="end"
                 android:lineSpacingExtra="@dimen/session_title_line_spacing"
-                android:lines="2"
+                android:maxLines="2"
```
Comment (konifar) [inline app/src/main/res/layout/item_horizontal_session.xml]: 👍

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -22,6 +22,7 @@
     <string name="session_fetch_failed">Failed to fetch session</string>
     <string name="session_content_description_speaker">Speaker image</string>
     <string name="session_content_description_favorite_button">Favorite button</string>
+    <string name="session_header_divider">" / "</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: 👍

**Comment by konifar** [comment]
LGTM! 💯 Thanks a lot!

**Pull request merged**